test/tests.cpp

```diff
@@ -1066,6 +1066,22 @@ TEST_CASE_METHOD(QInterfaceTestFixture, "test_sqrtx_reg")
     REQUIRE_THAT(qftReg, HasProbability(0, 8, 0x13));
 }
 
+TEST_CASE_METHOD(QInterfaceTestFixture, "test_sqrtxconjt")
```

Comment:
Can you add some validation in the middle of each of the sequence to make sure that the two implementations actually do something?  If they both were not implemented, this test would pass.

Reply:
I found it tricky to isolate the logic of these gates. However, I'll experiment and add something.